ext/mixed/js/display.js

```diff
@@ -265,6 +265,7 @@ class Display {
             $('.action-view-note').click(this.onNoteView.bind(this));
             $('.action-play-audio').click(this.onAudioPlay.bind(this));
             $('.kanji-link').click(this.onKanjiLookup.bind(this));
+            $('.glossary li').click(this.onTermLookup.bind(this));
```

Comment:
This doesn't work when there's only one glossary item, use `.glossary-item` instead